core/kogitobuild/buid_config_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Red Hat, Inc. and/or its affiliates
```

Comment:
This file name has a minor typo. :sweat_smile:

Reply:
Nice catch Kevin, do you mind fixing it?

Reply:
Sent PR's for [`master`](https://github.com/kiegroup/kogito-operator/pull/923) and [`1.5.x`](https://github.com/kiegroup/kogito-operator/pull/922) branch.